docs: add note to vercel isr on prerender page options conflict (#12429)
sveltejs/kit: documentation/docs/25-build-and-deploy/90-adapter-vercel.md

The `expiration` property is required; all others are optional.

> > Do not specify [prerender page options](/docs/page-options#prerender) on ISR routes as they take precedence over ISR configuration.

Reviewer: ```suggestion
> Do not specify [prerender page options](https://kit.svelte.dev/docs/page-options#prerender) on ISR routes as they take precedence over ISR configuration.
```

> > Do not specify [prerender page options](/docs/page-options#prerender) on ISR routes as they take precedence over ISR configuration.

Reviewer 2: ```suggestion
> Pages that are  [prerendered](/docs/page-options#prerender) will ignore ISR configuration.
```
